### ISSUE TITLE
Provisioner Create Pac throws an error

### ISSUE THREAD

**Issue by jswright61** [dyoung522/vmproxy#2]
When upgrading to the latest version.

``` bash
    >$ vagrant halt
    >$ git pull
    >$ vagrant up
```

…Extraneous output omitted

``` bash
    ==> default: Running provisioner: Create PAC (shell)...
        default: Running: /var/folders/hy/wm6746w10ds4fk9lrndz5zs80000gn/T/vagrant-shell20150217-77707-4yrawv.rb
    ==> default: stdin: is not a tty
    ==> default: /tmp/vagrant-shell:30:in `initialize'
    ==> default: :
    ==> default: No such file or directory - /usr/share/nginx/html/proxy.pac
    ==> default:  (
    ==> default: Errno::ENOENT
    ==> default: )
    ==> default:    from /tmp/vagrant-shell:30:in `open'
    ==> default:    from /tmp/vagrant-shell:30:in `<main>'
    The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by dyoung522** [comment]
Hmmm... Brian got the same error when he pulled the latest release. The fix for him was to clone a fresh copy, but I'll look into what may be causing this in case it happens to others.

**Comment by dyoung522** [comment]
fixed in d075bddc6d06fa85fd040f571e03d177430a3d26
